executor.go

```diff
@@ -100,6 +112,32 @@ func (r *executor) Execute(in ExecuteInput) error {
 	return nil
 }
 
+func (r *executor) validate(in ExecuteInput) error {
+	for _, task := range in.Tasks {
+		if task == "" {
+			return errors.New("task name cannot be empty")
+		}
+		if _, ok := r.defined[task]; !ok {
+			return errors.New("task provided but not defined: " + task)
+		}
+	}
+
+	if len(in.Tasks) == 0 {
+		return errors.New("no task provided")
+	}
```

Comment:
I know this code was moved from different place, but as default task is handled outside of `validate` this check could go before the loop.

Reply:
https://github.com/goyek/goyek/pull/420/commits/d83ead003b443456e223d43440ee7c563411028f